runner.js

```diff
@@ -2,6 +2,14 @@ const app = require('./app');
 
 const bookingDownloadUrl = 'https://indicators.hscic.gov.uk/download/PHF10/Data/BOOK_CANCEL_APPOINTMENTS_POMI.csv';
 const scriptsDownloadUrl = 'https://indicators.hscic.gov.uk/download/PHF10/Data/ORDER_REPEAT_PRESCRIPTIONS_POMI.csv';
+const codedRecordsDownloadUrl = 'https://indicators.hscic.gov.uk/download/PHF10/Data/DETAILED_CODED_RECORDS_POMI.csv';
 
-app.downloadAndProcessFile({ type: 'SCRIPTS', url: scriptsDownloadUrl });
-app.downloadAndProcessFile({ type: 'BOOKING', url: bookingDownloadUrl });
+const downloads = [
+  { type: 'BOOKING', url: bookingDownloadUrl },
+  { type: 'SCRIPTS', url: scriptsDownloadUrl },
+  { type: 'RECORDS', url: codedRecordsDownloadUrl },
+];
+
+downloads.forEach((request) => {
+  app.downloadAndProcessFile(request);
+});
```

Comment:
These three downloadAndProcessFile actions are running concurrently, as they are async tasks that are being started.

 I'm not sure it really matters at the moment as there's only three, but if we start downloading 10 things at once in the future it might slow things down - there'll be a pause while files are downloaded, followed by a series of CPU intensive tasks.